lib/util.js

```diff
@@ -993,6 +997,29 @@ function formatPromise(ctx, value, recurseTimes, keys) {
   return output;
 }
 
+function formatNamespaceProperty(ctx, ns, recurseTimes, key, array) {
+  let name;
+  let value;
+  try {
+    value = formatValue(ctx, ns[key], recurseTimes, array === 0);
+  } catch (err) {
+    if (err instanceof ReferenceError) {
+      value = ctx.stylize('<uninitialized>', 'special');
+    } else {
+      throw err;
+    }
+  }
+  if (typeof key === 'symbol') {
+    name = `[${ctx.stylize(key.toString(), 'symbol')}]`;
+  } else if (keyStrRegExp.test(key)) {
+    name = ctx.stylize(key, 'name');
+  } else {
+    name = ctx.stylize(strEscape(key), 'string');
+  }
+
+  return `${name}: ${value}`;
```

Comment:
Not a fan of copy pasting this part. It could easily get out of sync if anything small changes.